### PR TITLE
Export Draggable components from Table

### PR DESCRIPTION
**Pull request by themadcreator**
#### PR checklist

- [x] Addresses an existing issue: #251

#### What changes did you make?

Adds exports for Draggable and DragSelectable (and
supporting interfaces) to public API.

### CODE REVIEW

File: packages/table/src/index.ts [hunk 1]
```diff
@@ -38,10 +38,17 @@ export {
...
     Rect,
-    Utils
+    Utils,
 } from "./common/index";
 
 export {
+    ICoordinateData,
+    IDragHandler,
+    IDraggableProps,
+    Draggable,
```
Comment (cmslewis) [inline packages/table/src/index.ts]: nit: should these be alphabetized?

File: packages/table/src/index.ts [hunk 2]
```diff
@@ -51,24 +58,30 @@ export {
...
+    IDragSelectableProps,
+    DragSelectable,
+} from "./interactions/selectable";
+
+export {
     ColumnHeaderCell,
     IColumnHeaderCellProps,
     IColumnHeaderRenderer,
-    HorizontalCellDivider
+    HorizontalCellDivider,
```
Comment (cmslewis) [inline packages/table/src/index.ts]: Same

File: packages/table/src/index.ts [hunk 3]
```diff
@@ -51,24 +58,30 @@ export {
...
 export {
     IRowHeaderCellProps,
-    RowHeaderCell
+    RowHeaderCell,
 } from "./headers/rowHeaderCell";
 
 export {
     IEditableNameProps,
-    EditableName
+    EditableName,
```
Comment (cmslewis) [inline packages/table/src/index.ts]: Same

File: packages/table/src/index.ts [hunk 4]
```diff
@@ -51,24 +58,30 @@ export {
...
     IResizeHandleProps,
     Orientation,
-    ResizeHandle
+    ResizeHandle,
 } from "./interactions/resizeHandle";
 
 export {
+    ISelectableProps,
+    IDragSelectableProps,
+    DragSelectable,
```
Comment (cmslewis) [inline packages/table/src/index.ts]: Same

File: packages/table/src/index.ts (hunk 1 above)
Comment (adidahiya) [inline packages/table/src/index.ts]: huh I would hold off on that until we have an `ordered-exports` lint rule

**Comment by themadcreator** [comment]
We probably have the alpha sort import errors throughout the codebase, so lets' cross that bridge when we enable the linter rule.

**Pull request merged**